website/project/model.py

```diff
@@ -90,9 +90,7 @@ def validate_user(value):
     OSFUser = apps.get_model('osf.OSFUser')
     if value != {}:
         user_id = value.iterkeys().next()
-        try:
-            OSFUser.objects.get(_id=user_id)
-        except OSFUser.DoesNotExist:
+        if not OSFUser.objects.get(guids___id=user_id).exists():
```

Comment:
This should be `filter` instead of `get`. 

It would be great to have a regression test for this function.

Reply:
Yeahh, definitely gonna add a test.

Reply:
Wait, this function isn't used anywhere. Removing.

Reply:
`validate_doi` and `validate_category` are also unused. Removing those as well.